.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+on: [push, pull_request]
+
+jobs:
+  android-bin:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: seanmiddleditch/gha-setup-ninja@v3
+      - run: bazel build -c opt :encoder_main --config=android_arm64
+      - run: bazel build -c opt :decoder_main --config=android_arm64
+
+  android-app:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: seanmiddleditch/gha-setup-ninja@v3
```

Comment:
Ninja is unfortunatly not pre-installed. There are were multiple issues filed over at [actions/virtual-environments](https://github.com/actions/virtual-environments) (https://github.com/actions/virtual-environments/issues/514, https://github.com/actions/virtual-environments/issues/741, https://github.com/actions/virtual-environments/issues/1531 and https://github.com/actions/virtual-environments/discussions/2275), but unfortunately they won't budge.

I pinned the exact revision (warning: pinning could increase the maintenance burden it needs to be updated periodically).

Reply:
Thanks!  I don't see a step that depends on ninja here though.  Is it actually required?

Reply:
Ah it's only needed for the Linux build, now I understand. I assumed otherwise from the `sudo apt install ninja-build git cmake clang python` line in the [Common setup](https://github.com/google/lyra#common-setup) section of the Readme. I have removed the Ninja setup-jobs for the Android jobs!